ntoskrnl/ex/sysinfo.c

```diff
@@ -2363,9 +2363,8 @@ QSI_DEF(SystemSessionProcessesInformation)
 /* Class 54 - Load & map in system space */
 SSI_DEF(SystemLoadGdiDriverInSystemSpaceInformation)
```

Comment:
> name is actually `SystemLoadGdiDriverInSystemSpace` according to Geoff

Should we rename it?
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=SystemLoadGdiDriverInSystemSpaceInformation

Reply:
Indeed. Seems it makes sense.

Reply:
I have a full update on the whole enum up to Win 11 in my repo. I can PR it, once this is merged.

Reply:
Sounds like a plan